hamilton/parameter_analysis/merge_best_results.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pandas as pd
+import sys
+
+
+if __name__ == "__main__":
+    dataframes = []
+
+    for d in os.listdir(sys.argv[1]):
+        full_path = os.path.join(sys.argv[1], d)
+        if os.path.isdir(full_path) and not d.endswith("corrupted"):
+            f = os.path.join(full_path, "lowest_to_highest_mses.csv")
+            if os.path.isfile(f):
+                df = pd.read_csv(f, sep=",")
+                df = df.loc[(df["mean_squared_error"] < 3.5)]
```

Comment:
Could 3.5 be configurable through a variable, say, called quality_threshold that has a default value of 3.5?